src/main/java/me/partlysanestudios/partlysaneskies/garden/VisitorLogbookStats.kt

```diff
@@ -0,0 +1,158 @@
+/* VisitorLogbookStats.kt
+ * A Kotlin class written by Erymanthus[#5074] | (u/)RayDeeUx
```

Comment:
also you can remove your discord discriminator lol

Reply:
it's my github display name 😭

Reply:
Why does your github name have a discriminator lol